azurerm/internal/services/compute/tests/linux_virtual_machine_resource_other_test.go

```diff
@@ -521,6 +521,46 @@ func TestAccLinuxVirtualMachine_otherEncryptionAtHostEnabledWithCMK(t *testing.T
 	})
 }
 
+func TestAccLinuxVirtualMachine_otherSkipShutdownDefault(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_linux_virtual_machine", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: checkLinuxVirtualMachineIsDestroyed,
+		Steps: []resource.TestStep{
+			{
+				Config: testLinuxVirtualMachine_otherSkipShutdownDefault(data),
+				Check: resource.ComposeTestCheckFunc(
+					checkLinuxVirtualMachineExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "skip_shutdown", "true"),
+				),
+			},
+			data.ImportStep("skip_shutdown"),
```

Comment:
by moving this to the features block this is no longer necessary:

```suggestion
			data.ImportStep(),
```